admin-dev/themes/new-theme/js/components/form/delta-quantity-input.ts

```diff
@@ -87,10 +87,14 @@ export default class DeltaQuantityInput {
       const $updateElement = $container.find(this.config.updateQuantitySelector);
       $updateElement.toggleClass(this.config.modifiedQuantityClass, deltaQuantity !== 0);
     });
+
+    $(document).on('keyup', `${this.config.containerSelector} ${this.config.deltaInputSelector}`, (event) => {
```

Comment:
this part was added because now if you write something in the input it doesn't change until you press somewhere else outside the input. With this change it automatically updated on each input

Reply:
Ok, then we should probably extract the code from the change callback into a method that can be called for both events, instead of forcibly triggering the change event.

Either that or, maybe simpler, we can init the initial callback this way:
```js
$(document).on('change keyup', `${this.config.containerSelector} ${this.config.deltaInputSelector}`, (event: ChangeEvent) => {
```